var/lib/delphix-platform/ansible/apply

```diff
@@ -29,6 +29,20 @@ function find_playbooks() {
 		echo "Failure when finding playbooks." 2>&1
 		exit 1
 	fi
+
+	#
+	# Temporarily, we need to maintain backwards compatibility with
+	# software that still installs ansible configuration into the
+	# old directory. Once all consumers can be updated to install
+	# into the new directory, we can remove this logic.
+	#
+	if ! find "/etc/delphix-platform/ansible" \
+		-maxdepth 2 \
+		-mindepth 2 \
+		-name playbook.yml | sort -n; then
```

Comment:
This will result in 2 lists that are sorted locally, i.e. everything in `/etc/delphix-platform/ansible` will be executed after DLPX_ANSIBLE_DIRECTORY. Is this going to cause issues?

Reply:
> Is this going to cause issues?

No.

Long term, virtualization will get updated to install in `/var/lib/delphix-platform` and we'll remove this backwards compatibility layer.

Short term, the delphix-platform logic will be in `/var/lib/delphix-platform` using a prefix of `10-`, and the virtualization logic will be in `/etc/delphix-platform` using a prefix of `20-`. So, currently delphix-platform configuration is run first, while virtualization configuration is run second; this is due to their respective prefix numbers (`10-` and `20-`). Thus, the ordering of this won't practically be a problem, since we'll continue to execute the same configuration in the same order.

That is, unless I'm overlooking something.